app/javascript/comments.vue

```diff
@@ -150,6 +152,20 @@ export default {
     changeActiveTab(tab) {
       this.tab = tab
     },
+    displayMoreComments: function () {
```

Comment:
すみません、最初に気付くことができていればよかったのですが、他のメソッドの書き方が省略記法になっているのでここも合わせておくと良さそうです。
```suggestion
    displayMoreComments() {
```

Reply:
@yocajii 
ありがとうございます！修正しました。確認お願いします🙏